src/components/map/PositionArrow.tsx

```diff
@@ -1,34 +1,29 @@
 import React from 'react';
-import {TouchableOpacity, View} from 'react-native';
 import {CurrentLocationArrow} from '../../assets/svg/icons/places';
 import {StyleSheet} from '../../theme';
+import insets from '../../utils/insets';
+import Button from '../button';
 import shadows from './shadows';
-import ThemeIcon from '../theme-icon';
 
 const PositionArrow: React.FC<{flyToCurrentLocation(): void}> = ({
   flyToCurrentLocation,
 }) => {
   const styles = useStyles();
   return (
-    <TouchableOpacity
+    <Button
+      type="compact"
+      mode="primary"
       accessibilityLabel="Min posisjon"
       accessibilityRole="button"
       onPress={flyToCurrentLocation}
-    >
-      <View style={styles.flyToButton}>
-        <ThemeIcon svg={CurrentLocationArrow} />
-      </View>
-    </TouchableOpacity>
+      hitSlop={insets.symmetric(12, 20)}
```

Comment:
Wonder if we could make hitslops all over a themed value instead (just thinking out loud, not suggesting that you need to implement it here)

Reply:
Yeah, worth a discussion. If there are any drawpacks. There can be cases where buttons are closely aligned for instance.